syntax = "proto3";

package weather;

message WeatherRequest {
    bool end = 1;
}

message WeatherReply {
    float temperature = 1;
}

service Checker {
    rpc check_weather (WeatherRequest) returns (WeatherReply) {}
    rpc stream_weather (stream WeatherRequest) returns (stream WeatherReply) {}
}
